src/main/app/README.md

```diff
@@ -2,25 +2,29 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.7.
 
+## Type safety
+
+Please read about our [end-to-end type safety](../../../docs/development/endToEndTypeSafety.md) in this Angular app with openapi-typescript, aligning frontend and backend. The ZacHttpClient ensures type-safe requests and responses for better debugging.
+
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `npm run dev` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
 ## Code scaffolding
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+From the folder you want the scaffolding done, run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
 ## Build
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build. Run `npm run start` to run the built version.
 
 ## Running unit tests
 
-Run `ng test` to execute the unit tests via [Jest](https://jestjs.io/).
+Run `npm run test` or `npm run test:watch` to execute the unit tests via [Jest](https://jestjs.io/).
 
 ## Running end-to-end tests
```

Comment:
I would remove this entire section. We already have docs on how to run the e2e tests. Have nothing to do with this frontend Angular folder anyway.

Reply:
Removed